README.md

```diff
@@ -87,6 +87,12 @@ To add all of the components' processed and minified styles, reference `@carbon/
 - [Migration guides](docs/migration)
   - [`v1` to `v2`](docs/migration/migrate-to-2.x.md)
 
+## Tutorial
+
+A guide to create a React app with the Carbon Design System.
+
+[Carbon Developer Essentials - React Tutorial](https://www.carbondesignsystem.com/tutorial/react/overview)
```

Comment:
```suggestion
## Carbon Developer Essentials

If you are new to the Carbon Design System, we recommend that you take the [Carbon Developer Essentials - React Tutorial](https://www.carbondesignsystem.com/tutorial/react/overview). The tutorial shows you how use the Carbon Design System to build a small React application, and you will receive a badge upon completion and approval.

Reply:
(Please note that this is a multi-line suggestion that starts at line 90 😅 )

Reply:
I'll pull in when I have some free time and incorporate :)

Reply:
@b3nk3 using the code suggestion above, you can click on the "commit suggestion" button, write a commit message, and then commit changes directly to this branch from the GitHub UI. 👍

Reply:
yeah I figured... was unsure if it will have the whole suggestion, but it did indeed ;)